src/app/pages/applications/dialogs/chart-upgrade/chart-upgrade-dialog.component.scss

```diff
@@ -0,0 +1,96 @@
+:host {
+  font-size: 14px !important;
```

Comment:
Please do your best to try using more specificity to avoid using `!important` here or below.